apps/simple-trading-app/src/app/hooks/use-settlement-account.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useSettlementAccount } from './use-settlement-account';
+import type { PartyBalanceQuery_party_accounts } from '../components/deal-ticket/__generated__/PartyBalanceQuery';
+import { render, screen } from '@testing-library/react';
+
+interface Props {
+  settlementAssetId: string;
+  accounts: PartyBalanceQuery_party_accounts[];
+}
+
+const MockComponent = ({ settlementAssetId, accounts }: Props) => {
+  const settlementAccount = useSettlementAccount(settlementAssetId, accounts);
```

Comment:
suggestion (non blocking): We are already using `@testing-library/react-hooks` if you want to use it!

Reply:
Nice! Will refactor.